plugins.go

```diff
@@ -217,10 +217,12 @@ func RegisterPlugin(name string, plugin Plugin) {
 // EventName represents the name of an event used with event hooks.
 type EventName string
 
-// Define the event names for the startup and shutdown events
+// Define the event names for the startup, shutdown, instancestartup and certrenew events
```

Comment:
Might as well just change this to "names for the various events", it'll get pretty wordy if you keep adding each one to the comment every time you set up a new event 😆

Reply:
Right 😆